subt/zmq-subt-x4.json

```diff
@@ -13,6 +13,9 @@
             "virtual_bumper_radius": 10.0,
             "virtual_world": true,
             "speed_policy": "conservative",
+            "rotation_p": 1.6,
+            "turbo_safe_dist": 3.2,
+            "turbo_speed": 2.3,
```

Comment:
In fact, X4 flies so nicely that we may try higher speed. Sky is the limit. Resp. [5 m/s](https://github.com/osrf/subt/blob/c6f1deaeaaa672d863b98f738ebdbd1382e14b38/submitted_models/ssci_x4_sensor_config_2/launch/spawner.rb#L170). Up to you if you are in a mood to do that or we leave it for later.

Reply:
If you have suggestion of reasonable parameters for faster motion, please let me know? Or you would just increase `turbo_speed` to 5.0? I would maybe vote for merge as it is and then create new experiment with even higher values ... we will see later today.

Reply:
p.s. I guess we should increase also the `turbo_safe_dist` but then it may turn out that we will not turn turbo at all or rarely? (there is just IF, no proportional speed, right?)

Reply:
I wouldn't increase the safe distance. It is already close to the limit of what we get in the tunnel environments. 3.0 may be even better there now that X4 is better behaved.

So, what I would try: safe_dist = 3.0, turbo_speed in [3.0, 4.0, 5.0], looking for overshoots when turning left/right and for hitting walls, especially at the end of the tunnel. Halving intervals from there on. After finding the highest speed where it works, I would go 0.3 to 0.5 m/s lower to give us some safety margin.

All this may be faster done when running it locally, so merging and iterating later sounds good to me too.

Reply:
OK, then I would merge this PR and create new one for experiments - in particular I am wondering how this will behave in other worlds and also how it will "collide" with octomap. Thanks m.